media-libs/libdiscid/libdiscid-0.6.2.recipe

```diff
@@ -4,14 +4,15 @@ disc IDs from audio CDs. It reads a CD's table of contents (TOC) and \
 generates an identifier which can be used to lookup the CD at MusicBrainz. \
 Additionally, it provides a submission URL for adding the disc ID to the \
 database and gathers ISRCs and the MCN (=UPC/EAN) from disc."
-HOMEPAGE="http://musicbrainz.org/doc/libdiscid"
-COPYRIGHT="2012 Andrew Hawkins"
+HOMEPAGE="https://musicbrainz.org/doc/libdiscid"
+COPYRIGHT="2017 Robert Kaye et al"
 LICENSE="GNU LGPL v2.1"
-REVISION="2"
+REVISION="3"
 SOURCE_URI="http://ftp.musicbrainz.org/pub/musicbrainz/libdiscid/libdiscid-$portVersion.tar.gz"
 CHECKSUM_SHA256="f9e443ac4c0dd4819c2841fcc82169a46fb9a626352cdb9c7f65dd3624cd31b9"
+PATCHES="libdiscid-$portVersion.patchset"
 
-ARCHITECTURES="x86_gcc2 x86 x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
```

Comment:
Since you are fixing this recipe, could you please also make this small change?
```diff
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
 	libdiscid$secondaryArchSuffix = $portVersion
-	lib:libdiscid$secondaryArchSuffix = 0.6.2 compat >= 0
+	lib:libdiscid$secondaryArchSuffix = $libVersionCompat
 	"
```
And a few lines below:
```diff
 PROVIDES_devel="
 	libdiscid${secondaryArchSuffix}_devel = $portVersion
-	devel:libdiscid$secondaryArchSuffix = 0.6.2 compat >= 0
+	devel:libdiscid$secondaryArchSuffix = $libVersion
 	"
```
(I can't attach any comment to the PROVIDES_devel bloc because it is not near a modified line.)

@alaviss, what would you put here, for the PROVIDES_devel? Would you use `$libVersion` or `$libVersionCompat`?

Reply:
I would use `$libVersionCompat` as the library already provides ABI stability, so there's no doubt that the API is stable as well